nssp_demo/batch/setup_pool.py

```diff
@@ -36,6 +36,7 @@ def main(pool_name: str) -> None:
     pool_config.mount_configuration = blob.get_node_mount_config(
         storage_containers=[
             "nssp-etl",
```

Comment:
Is nssp-etl used at all here?

Reply:
Yes, this tells Batch that when nodes are spun up in this pool they should mount this blob storage container, which is useful because it contains the (later) vintages.